pandas/tests/io/test_rdata.py

```diff
@@ -0,0 +1,565 @@
+from io import BytesIO
```

Comment:
Need to add a test where a reader and a writer  is pickled and unpickled to make sure the Cython classes work as expected aroudn pickling.

Reply:
Can you explain a little more here? Can you point me to a working example of such pickling?

Reply:
Pickling of non-trivial cython classes can require an implementation of __setstate__ and __reduce__.  You can write the test to see if it works.

1. Create Reader or Writer
2. Pickle to string using dumps and reload using loads
3. Use the unpickled Reader or Writer to read or write a file.

If the test passes, and the unpickled reader/writer perform the same as a fresh instance, then you are OK.